playgrounds/skeleton-svelte/src/routes/components/steps/+page.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { Steps } from '@skeletonlabs/skeleton-svelte';
+</script>
+
+<Steps defaultStep={0} count={3}>
+	<Steps.List>
+		<Steps.Item index={0}>
+			<Steps.Trigger>
+				<Steps.Indicator>1</Steps.Indicator>
+				<Steps.ItemText>First</Steps.ItemText>
+			</Steps.Trigger>
+			<Steps.Separator />
+		</Steps.Item>
+		<Steps.Item index={1}>
+			<Steps.Trigger>
+				<Steps.Indicator>2</Steps.Indicator>
+				<Steps.ItemText>Then</Steps.ItemText>
+			</Steps.Trigger>
+			<Steps.Separator />
+		</Steps.Item>
+		<Steps.Item index={2}>
+			<Steps.Trigger>
+				<Steps.Indicator>3</Steps.Indicator>
+				<Steps.ItemText>Finally</Steps.ItemText>
+			</Steps.Trigger>
+		</Steps.Item>
+	</Steps.List>
+
+	<Steps.Content index={0}>First do this.</Steps.Content>
+	<Steps.Content index={1}>Then do that.</Steps.Content>
+	<Steps.Content index={2}>Almost there...</Steps.Content>
+	<Steps.Content index={3}>All done!</Steps.Content>
+
+	<Steps.Control>
```

Comment:
Let's follow Zag's lead here and drop this component part in favor of a user-provided `<div>`. Feel free to use these classes to position the buttons on the left/right:

```html
<div class="flex justify-between items-center gap-2">...</div>
```

Reply:
removed Steps.Control